lib/krane/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module Krane
-  VERSION = "1.1.4"
+  VERSION = "1.1.5"
```

Comment:
Generally we do version bumps atomically (that is, a PR with _only_ the version change and no logic changes). Let's set this back to 1.1.4 and once this merges we can make the version bump PR

Reply:
Will do!

Reply:
I assume this also includes the CHANGELOG?

Reply:
>I assume this also includes the CHANGELOG?

You can keep the changelog entries (it makes it easier since we won't have to backfill them during the version bump)

Reply:
👍